src/components/FilmsList.css

```diff
@@ -0,0 +1,32 @@
+.divStyle {
+  display: block;
+  width: -webkit-fill-available;
+  height: fit-content;
+  background-color: white;
+  border: 1px solid;
+  padding: 5px;
+}
+.leftSide {
+  display: inline;
+  float: left;
+}
+.rightSide {
```

Comment:
Look for in Google "how to name classes in CSS"

Reply:
right-side
we use in CSS kebab-case, not camelCase (Java, JS, ...)